04-concept-api/00-overview.md

```diff
@@ -17,9 +17,46 @@ Anything in Grakn, whether a concept type or a data instance, is a [Concept](../
 
 **Rule** refers to a [Graql Rule](../09-schema/03-rules.md).
 
+## Local and Remote Concept API
+
+The Concept API architecture is implemented in 2 ways in each client: **local** and **remote**. The concepts share the same overall structure, but only a small set of methods are available on the *local* concepts.
+
+The benefit of using local concepts is that time spend waiting for round-trips to the server is eliminated: all of the information required is fetched in one go during the query. Calls to methods on the *remote* concept API must make at least one full round-trip to the server to fetch a result, which will include at least the network latency between your client and the Grakn server.
+
+<div class="note">
+As an example, if you were to make 1,000 calls method calls Remote Concept API and your latency (ping) between client is 10ms, you would spend a total of 10 seconds waiting for network round-trips.
+
+Where efficiency is a concern, especially when dealing with large numbers of results, we recommend that you include the required information to fetch in the query, rather than using the Remote Concept API.
+</div>
+
+In order to make use of this benefit, *queries always return local concepts* so that the pre-fetched information is available. Since the behaviour of remote methods is slightly different and involves binding, *local* concepts must be converted to *remote* concepts before remote methods are available. Please read the following sections for more information on how to convert local concepts to remote ones.
+
+### Local Concept API
+
+For the *local* Concept API, each concept must contain all of the information that can be returned from the concept API methods from the point that it is created. This information will not be updated as the concept changes on the server, so it may become out of date once the transaction has ended or if changes have been made during the transaction. Local concepts are not bound to a transaction, so they can be safely used even after the transaction, session or client has been closed. This also means that **local concepts can never return remote concepts**: if you are using local concepts, you are guaranteed that no additional network operations will be made and therefore that no blocking is necessary.
```

Comment:
the bold bit reads a bit like remote concepts might be able to return local concepts? I don't think that you need to specify it further and we can probably delete the last sentence or make it much shorter - "local concepts never perform network operations"